lib/msprime.c

```diff
@@ -1620,6 +1622,7 @@ msp_recombination_event(msp_t *self)
     self->num_re_events++;
     /* We can't use the GSL integer generator here as the range is too large */
     l = 1 + (int64_t) (gsl_rng_uniform(self->rng) * (double) num_links);
+    printf( "l = %d, num_links = %d\n", (int)l, (int)num_links);
```

Comment:
Well you need to comment it out it before submitting CI builds --- it's spewing out many megabytes of text and causing the builds to fail before they even finish running.